src/service/formSubmitHandler.js

```diff
@@ -9,16 +10,67 @@ const formSubmitHandler = async (url, user, authDispatch) => {
     return new Promise(resolve => {
       if (response.status === 200) {
         resolve(response.status);
+        toast.success(`Welcome ${response.data.foundUser.firstName} `, {
+          position: 'top-right',
+          autoClose: 5000,
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: true,
+          draggable: true,
+          progress: undefined,
+        });
         authDispatch({ type: 'LOGIN', payload: response.data });
       } else if (response.status === 201) {
+        console.log(response);
         resolve(response.status);
+        toast.success(`Welcome ${response.data.createdUser.name} `, {
+          position: 'top-right',
+          autoClose: 5000,
+          hideProgressBar: false,
+          closeOnClick: true,
+          pauseOnHover: true,
+          draggable: true,
+          progress: undefined,
+        });
         authDispatch({ type: 'SIGNUP', payload: response.data });
       } else {
         resolve(response.status);
       }
     });
   } catch (err) {
-    console.error(err);
+    if (err.response.status === 422) {
+      toast.error(`Email Already exist !`, {
+        position: 'top-right',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+    }
+    if (err.response.status === 401) {
+      toast.error(`The credentials are invalid`, {
+        position: 'top-right',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+    }
+    if (err.response.status === 404) {
+      toast.error(`The email you entered is not Registered !`, {
+        position: 'top-right',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+    }
```

Comment:
if you do `err.response.data.errors[0]` you will get the error message which is written in backend, which you can put it in toast messages. Try console.logging the line above, and then you can directly add the message in toast instead of writing different cases

Reply:
Thank you for the suggestion.